### PR TITLE
Adds docs for jobs setup and generate commands

### LINKED ISSUE #11357: Update CLI docs with new jobs commands

### PR DESCRIPTION
**Pull request by cannikin**
Closes #11357

### CODE REVIEW

File: docs/docs/cli-commands.md [hunk 1]
```diff
@@ -436,7 +436,8 @@ Some generators require that their argument be a model in your `schema.prisma`.
 | `dataMigration <name>` | Generate a data migration component                                                                   |
 | `dbAuth`               | Generate sign in, sign up and password reset pages for dbAuth                                         |
 | `deploy <provider>`    | Generate a deployment configuration                                                                   |
-| `function <name>`      | Generate a Function                                                                                   |
+| `function <name>`      | Generate a Job                                                                                        |
```
Comment (Tobbe) [inline docs/docs/cli-commands.md:439]:
```suggestion
| `function <name>`      | Generate a Function                                                                                        |
```

File: docs/docs/cli-commands.md [hunk 2]
```diff
@@ -735,6 +736,35 @@ $ /redwood-app/node_modules/.bin/dev-server
...
+
+```bash
+yarn redwood generate job <name>
+```
+
+| Arguments & Options  | Description                                                                           |
+| -------------------- | ------------------------------------------------------------------------------------- |
+| `name`               | Name of the job ("Job" suffix is optional)                                            |
+| `--force, -f`        | Overwrite existing files                                                              |
+| `--typescript, --ts` | Generate TypeScript files. Enabled by default if we detect your project is TypeScript |
```
Comment (Tobbe) [inline docs/docs/cli-commands.md:751]: "is using TypeScript" maybe? or "is a TypeScript project"?

Comment (cannikin) [reply docs/docs/cli-commands.md:751]: I copied from one of the other ones, so we'll probably have a few more to change!

Comment (Tobbe) [reply docs/docs/cli-commands.md:751]: Meh. Leave it then

File: docs/docs/cli-commands.md (hunk 1 above)
Comment (cannikin) [reply docs/docs/cli-commands.md]: Bah, text wrapping confused me, should be fixed

**Review by Tobbe — APPROVED** [review]

**Pull request merged**